Neuro/Minigames/Solvers/AlignTelescopeSolver.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+using UnityEngine;
+
+namespace Neuro.Minigames.Solvers;
+
+[MinigameSolver(typeof(TelescopeGame))]
+public sealed class AlignTelescopeSolver : GeneralMinigameSolver<TelescopeGame>
+{
+    public override IEnumerator CompleteMinigame(TelescopeGame minigame, NormalPlayerTask task)
+    {
+        yield return InGameCursor.Instance.CoMoveTo(minigame.Reticle);
+        InGameCursor.Instance.StartHoldingLMB(minigame);
+
+        // Add a bit of human
+        Vector2 rand = new Vector2(Random.RandomRange(-0.4f, 0.4f), Random.RandomRange(-0.4f, 0.4f));
+        float speed = Random.RandomRange(0.1f, 0.4f);
+
+        // Cursor controls background so it needs to be moved opposite of the target
+        yield return InGameCursor.Instance.CoMoveTo(InGameCursor.Instance.Position + -(Vector2)minigame.TargetItem.transform.localPosition + rand, speed);
```

Comment:
I don't like the idea of snapping directly to the minigame target, it doesn't feel very natural

Reply:
Updated to cycle through all items in a random sequence instead of directly going to target.

Reply:
The intention of the minigame solvers is to mimic how a human would solve the task. In this case, there are a few problems that I've noticed:
1. Since the order is random, it sometimes goes across the same object multiple times, it would be better to go in a circular motion
2. If the target is spotted on screen, the solver should snap to it

Reply:
https://github.com/VedalAI/neuro-amongus/pull/76/commits/cb536c6e594daf63d9cea0cb797676d7921f77f4
Updated to sweep a circle which covers all objects, snaps to target if in view